src/gui/src/SoundLibrary/SoundLibraryExportDialog.cpp

```diff
@@ -81,6 +85,28 @@ void SoundLibraryExportDialog::on_exportBtn_clicked()
 	QString drumkitDir = Filesystem::drumkit_dir_search( drumkitName );
 	QString saveDir = drumkitPathTxt->text();
 
+	int componentID = -1;
+	Drumkit* info;
+	if( versionList->currentText().toStdString() == "0.9.6 and lower" ) {
+		for (uint i = 0; i < drumkitInfoList.size(); i++ ) {
+			info = drumkitInfoList[i];
+			if( info->get_name().compare( drumkitName ) == 0 ) {
+				QString temporaryDrumkitXML = QString("/tmp/hydrogen/drumkit.xml");
```

Comment:
This won't work for systems which don't have a unix file system layout. There is a QT helper function which gives you an appropriate temporary path, we are using that to implement Preferences::get_instance()->getTmpDirectory()